File: lib/scripts/python/hello.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-

import sys

def main():
    """
    Hello
    """
    if len(sys.argv) <= 1:
      print('=============')
      print('Hello, Python')
      print('=============')
    elif len(sys.argv) > 1:
      print('')
      print('Hello, '+sys.argv[1])
      print('')

if __name__ == "__main__":
    main()

File: lib/scripts/perl/hello.pl
#!/usr/bin/perl -w
use strict;
use warnings;

if( $#ARGV < 0 ) {
  print("=============\n");
  print("Hello, Perl\n");
  print("=============\n");
} else {
  print("\n");
  print("Hello $ARGV[0] \n");
  print("\n");
}
